tests/conditional_compilation/conftest.py

```diff
@@ -76,9 +82,14 @@ def pytest_generate_tests(metafunc):
             extra_args["marks"] = test["marks"]
 
         test_id = model_path.replace('$', '').replace('{', '').replace('}', '')
-        params.append(pytest.param(test_id, Path(expand_env_vars(model_path)), **extra_args))
+        if 'libs' in metafunc.fixturenames:
```

Comment:
looks like an absolute hardcode which isn't too flexible for further development. Let's split common `pytest_generate_tests` to 2 functions each will be defined in a test .py files